src/datatypes/character/Character.hpp

```diff
@@ -15,137 +15,70 @@
 #include <util/UUID.hpp>
 #include <util/Point.hpp>
 
+constexpr unsigned int DEFAULT_HEALTH_POINTS = 100;
+constexpr unsigned int DEFAULT_CHIPS = 10;
+constexpr unsigned int DEFAULT_ACTION_POINTS = 0;
+constexpr unsigned int DEFAULT_MOVE_POINTS = 0;
+constexpr unsigned int DEFAULT_INTELLIGENCE_POINTS = 0;
```

Comment:
Ich sehe keinen guten Grund das hier auszulagern, und erst recht nicht das in globalem namespace zu machen.
Wenn, dann in `spy::character`

Reply:
Das Auslagern war dadurch bedingt, dass Sonar die in-class Initialisierung der initializer-list vorzieht und daraus sogar einen major code smell macht. Aber du hast recht, im Namespace ist das definitiv besser aufgehoben

Reply:
Jo in-class finde ich OK, fand nur die globalen konstanten hier ein bisschen unnötig